src/plays/states/States.jsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from "react";
+import { useLocation } from "react-router-dom";
+import { getPlayById } from "meta/play-meta-util";
+import PlayHeader from "common/playlists/PlayHeader";
+import "./States.css";
```

Comment:
usually the css file names starts with small case

Reply:
Rectified!